2022-12-04 18:19:01,193 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-12-04 18:19:01,194 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\joaom\AppData\Local\Programs\Python\Python310\lib\site-packages\instabot\bot\bot.py
2022-12-04 18:19:01,194 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-12-04 18:19:01,491 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2022-12-04 18:19:01,822 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-12-04 18:19:02,128 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-12-04 18:19:02,429 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-12-04 18:19:03,712 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-12-04 18:19:03,712 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https:\\/\\/i.instagram.com\\/challenge\\/AXEieKCJvq9z5gz-NEGdwrwis-AE07Tgnxo2eKaMxsIxN8k0mn0xiEDCX5PuS085CcESZGg\\/AfwYFmZeXvBSK_yBn6FVHpJfiNkpu4YlReKD4-E-SikO0maex90-TragfWPD5h2M6AXFIBWC-h4upA\\/","api_path":"\\/challenge\\/AXEieKCJvq9z5gz-NEGdwrwis-AE07Tgnxo2eKaMxsIxN8k0mn0xiEDCX5PuS085CcESZGg\\/AfwYFmZeXvBSK_yBn6FVHpJfiNkpu4YlReKD4-E-SikO0maex90-TragfWPD5h2M6AXFIBWC-h4upA\\/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required","exception_name":"ChallengeRequired"}'
2022-12-04 18:19:03,713 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-12-04 18:19:03,713 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-12-04 18:19:03,713 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
